src/Symfony/Component/Serializer/Context/ContextBuilderTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Context;
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ */
+trait ContextBuilderTrait
+{
+    /**
+     * @var array<string, mixed>
+     */
+    protected array $context;
```

Comment:
```suggestion
    ptivate array $context;
```

Reply:
Unfortunately, this must be protected because of abstract normalizes (such as `AbstractObjectNormalizer` for example)